packages/astro/src/assets/utils/remoteProbe.ts

```diff
@@ -37,8 +37,7 @@ export async function probe(url: string): Promise<ISize> {
 					return dimensions;
 				}
 			} catch (error) {
-				// This catch block is specifically for `sizeOf` failures,
-				// which might occur if the accumulated data isn't yet sufficient.
+				throw new Error(`Failed to determine the type or the size of the picture (${url}): ${error}`);
```

Comment:
This would break `inferSize` in some cases, the error is ignored on purpose here as the comment previously there said.

Reply:
done